### PR TITLE
[zh] Resync kubeadm implementation details

### PR DESCRIPTION
**Pull request by tengqm**
This PR resyncs changes to the implementation details page and converts the EOL style from DOS to UNIX.

### CODE REVIEW

File: content/zh-cn/docs/reference/setup-tools/kubeadm/implementation-details.md [hunk 1]
```diff
@@ -1,1286 +1,1341 @@
...
+- Flags for using certificates generated in previous steps:
+
+  - `--root-ca-file` to `ca.crt`
+  - `--cluster-signing-cert-file` to `ca.crt`, if External CA mode is disabled, otherwise to `""`
+  - `--cluster-signing-key-file` to `ca.key`, if External CA mode is disabled, otherwise to `""`
+  - `--service-account-private-key-file` to `sa.key`
+-->
+- 使用先前步骤中生成的证书的标志：
+
+  -`--root-ca-file` 设为 `ca.crt`
```
Comment (Sea-n) [inline content/zh-cn/docs/reference/setup-tools/kubeadm/implementation-details.md:751]:
```suggestion
  - `--root-ca-file` 设为 `ca.crt`
```

**Review by Sea-n — COMMENTED** [review]
/approve

File: content/zh-cn/docs/reference/setup-tools/kubeadm/implementation-details.md [hunk 2]
```diff
@@ -1,1286 +1,1341 @@
...
+  /etc/kubernetes/bootstrap-kubelet.conf. Inside this file there is a bootstrap-token or embedded
+  client certificates for authenticating this node with the cluster.
+
+  This client cert should:
+
+  - Be in the `system:nodes` organization, as required by the
+    [Node Authorization](/docs/reference/access-authn-authz/node/) module
+  - Have the Common Name (CN) `system:node:<hostname-lowercased>`
+-->
+- 供 kubelet 在 TLS 引导期间使用的 kubeconfig 文件 —— `/etc/kubernetes/bootstrap-kubelet.conf`。
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/setup-tools/kubeadm/implementation-details.md:406]: 查阅资料显示，破折号前后不需要加空格，下同（420 行和 432 行）
ref: https://people.ubuntu.com/~happyaron/l10n/GB(T)15834-2011.html

File: content/zh-cn/docs/reference/setup-tools/kubeadm/implementation-details.md [hunk 3]
```diff
@@ -1,1286 +1,1341 @@
...
+<!--  
+1. All images will be pulled from k8s.gcr.io by default.
+   See [using custom images](/docs/reference/setup-tools/kubeadm/kubeadm-init/#custom-images)
+   for customizing the image repository
+1. In case of kubeadm is executed in the `--dry-run` mode, static Pods files are written in a
+   temporary folder
+1. Static Pod manifest generation for control plane components can be invoked individually with
+   the [`kubeadm init phase control-plane all`](/docs/reference/setup-tools/kubeadm/kubeadm-init-phase/#cmd-phase-control-plane) command
+-->
+1. 所有镜像默认从 k8s.gcr.io 拉取。
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/setup-tools/kubeadm/implementation-details.md:536]:
```suggestion
1. 所有镜像默认从 k8s.gcr.io 拉取。关于自定义镜像仓库，
   请参阅[使用自定义镜像](/zh-cn/docs/reference/setup-tools/kubeadm/kubeadm-init/#custom-images)。
```

File: content/zh-cn/docs/reference/setup-tools/kubeadm/implementation-details.md [hunk 4]
```diff
@@ -1,1286 +1,1341 @@
...
+    to enforce default storage class on `PersistentVolumeClaim` objects
+  - [`DefaultTolerationSeconds`](/docs/reference/access-authn-authz/admission-controllers/#defaulttolerationseconds)
+  - [`NodeRestriction`](/docs/reference/access-authn-authz/admission-controllers/#noderestriction)
+    to limit what a kubelet can modify (e.g. only pods on this node)
+  -->
+  - [`DefaultStorageClass`](/zh-cn/docs/reference/access-authn-authz/admission-controllers/#defaultstorageclass)
+    在 `PersistentVolumeClaim` 对象上强制使用默认存储类型
+  - [`DefaultTolerationSeconds`](/zh-cn/docs/reference/access-authn-authz/admission-controllers/#defaulttolerationseconds)
+  - [`NodeRestriction`](/zh-cn/docs/reference/access-authn-authz/admission-controllers/#noderestriction)
+    限制 kubelet 可以修改的内容（例如，仅此节点上的 pod）
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/setup-tools/kubeadm/implementation-details.md:635]:
```suggestion
    限制 kubelet 可以修改的内容（例如，仅此节点上的 Pod）
```

File: content/zh-cn/docs/reference/setup-tools/kubeadm/implementation-details.md (hunk 2 above)
Comment (tengqm) [reply content/zh-cn/docs/reference/setup-tools/kubeadm/implementation-details.md:406]: 我态度中立。

**Pull request merged**